mirrord/config/src/external_proxy.rs

```diff
@@ -27,6 +27,18 @@ pub static MIRRORD_EXTERNAL_TLS_KEY_ENV: &str = "MIRRORD_EXTERNAL_TLS_KEY";
 #[config(map_to = "ExternalProxyFileConfig", derive = "JsonSchema")]
 #[cfg_attr(test, config(derive = "PartialEq"))]
 pub struct ExternalProxyConfig {
+    /// ### external_proxy.listen {#external_proxy-listen}
+    ///
+    /// Provide a specific address to listen to for external proxy
+    /// (will try and bind local machine ip if not specified)
+    pub listen: Option<IpAddr>,
+
+    /// ### external_proxy.address {#external_proxy-address}
```

Comment:
Same for this config. The user might have to set this up why?

Reply:
I updated it a bit but can change it to hostname sadly